app/src/main/res/menu/main_menu.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/main_menu_address_book"
+        android:icon="@drawable/ic_group_black_24dp"
```

Comment:
same as above, that is why we have `ic_menu_settings` which wraps `ic_settings`

Reply:
I removed the wrapper and it worked on older platforms also